src/Views/HardwareView.vala

```diff
@@ -233,6 +255,36 @@ public class About.HardwareView : Gtk.Grid {
         return result;
     }
 
+    private string? get_gpu_info (bool primary) {
+        string? gpu_name = null;
+
+        if (switcheroo_interface != null) {
+            foreach (unowned HashTable<string,Variant> gpu in switcheroo_interface.gpus) {
+                bool is_default = gpu.get ("Default").get_boolean ();
+
+                if (is_default == primary) {
+                    gpu_name = clean_name (gpu.get ("Name").get_string ());
+                }
+            }
+        }
+
+        if (gpu_name != null) {
+            return gpu_name;
+        }
+
+        // Switcheroo failed to get the name of the secondary GPU, we'll assume there isn't one
+        // and return null
+        if (!primary) {
+            return null;
+        }
+
+        if (session_manager != null) {
+            return clean_name (session_manager.renderer);
+        }
+
+        return _("Unknown GPU");
```

Comment:
Do we want something more meaningful like `_("Primary GPU")` or `_("Secondary GPU")` so that it doesn't look broken :thinking:

Reply:
I've just noticed I've changed the string from "Unknown Graphics" to "Unknown GPU" by accident. So I've changed that back.

But it's super unlikely that we'd hit this after querying two different sources for GPU information. I still think I'd rather see:

```
Quad-Core Intel Core i7-XXXX
Unknown Graphics
16 GB memory
128 GB storage
```

Instead of 
```
Quad-Core Intel Core i7-XXXX
Primary GPU/Graphics
16 GB memory
128 GB storage
```